### ISSUE TITLE
Collaboration Cycle for Authenticated Experience | Profile | CH33 EDU Direct Deposit Updates

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#76630]
### VFS team name

Authenticated Experience - Profile

### Product name

Profile

### Feature name

CH33 EDU Direct Deposit

### GitHub label for team

authenticated-experience

### GitHub label for product

profile

### Public DSVA Slack channel for VFS team

#edu-direct-deposit-migration   https://dsva.slack.com/archives/C06JV0KHZNF

## Kickoff questions
<details><summary>Toggle kickoff questions</summary>

### When did/will you start working on this product/feature?

2/20/2024

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Unsure

### Will your work involve changes to...

Tools, applications and dynamic pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

Yes

### Do you need to capture any additional analytics or metrics?

Unsure

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/direct-deposit/ch33-bdn-corpdb-migration

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Add the GitHub labels for your team, product, and feature to this ticket.

- [X] I acknowledge that I must add the GitHub labels for my team, product, and feature to this ticket.

### Notify the Collaboration Cycle team of this ticket in your team's public Slack channel.

- [X] I acknowledge that I must notify the Governance team by completing the Collab Cycle Kickoff workflow in my team's Slack channel after submitting this issue. This can be completed by typing "/collab cycle" and selecting the kickoff workflow.
- 
Kickoff Slack Thread with VFS team: [Kickoff thread](https://dsva.slack.com/archives/C06JV0KHZNF/p1708460001226429)
</details>

### Design Intent
<details>
  <summary>Toggle Design Intent instructions</summary>

##### VFS team actions
- [ ] Review [Design Intent Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent) to understand what this touchpoint involves.
- [ ] Schedule your Design Intent (with at least 2 business days lead time from now):
  - Open the [Calendly design intent calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
  - Select a date and time and click "Confirm"
  - Add your name and email
  - Click "Add Guests" and enter the email addresses for VFS team attendees
  - Click "Schedule Event"
- [ ] Notify the Collaboration Cycle team of this event by using the `/Collab Cycle Design Intent` slash-command in your team channel. Provide information as prompted by the Slack workflow.
- [ ] Link all artifacts in the `Design Intent artifacts for review` section below at least two business days before the scheduled Design Intent. Please don't add artifacts in the comments section.
  
**Design Intent artifacts for review**

[See guidance on Design Intent artifacts.](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent#Designintent-Artifacts) Governance Team feedback is based on the artifacts provided here as well as information provided during the meeting. Please provide links to artifacts **at least two business days** before the scheduled meeting.

Required:
- [ ] User flow

Not required, but nice to have:
- [ ] Wireframes

Optional:
- [ ] Research plan
- [ ] Any other artifacts you have so far
  
##### Governance Team actions
- [ ] Design Intent Slack thread with VFS team
- [ ] Meeting date/time:
  
##### Governance Team actions

- [ ] Update this ticket with the Zoom recording
  - Recording link
  - Passcode:
- Accessibility
  - [ ] Feedback ticket attached
  - [ ] No feedback
- Design
  - [ ] Feedback ticket attached
  - [ ] No feedback
- IA
  - [ ] Feedback ticket attached
  - [ ] No feedback
  
##### VFS team actions

- [ ] Review feedback tickets. Comment on the ticket if there are any questions or concerns.
  
</details>

### Sitewide CAIA Intake Request
<details>
  <summary>Toggle Sitewide CAIA Intake Request</summary>

##### VFS actions
- [ ] Complete the [Sitewide CAIA intake request process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Sitewide-CAIA-intake-request.2124906551.html)
- [ ] Link to the Sitewide CAIA Intake Request ticket below

</details>

### Research Review
<details>
  <summary>Toggle Research Review</summary>

#### VFS actions
- [ ] Complete the [research review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Research-plan-review.1781891143.html)

</details>

### Midpoint Review
<details>
  <summary>Toggle Midpoint Review</summary>
  
##### VFS actions
Navigate to reference link: [Midpoint Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review)  
- [ ] Schedule your Midpoint Review when ready:
   - Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [ ] Check this box if you'd like this review to be asynchronous (Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review)
- [ ] Notify the Collaboration Cycle team of this event by using the `/Collab Cycle Midpoint Review` "slash-command" in your team channel. Provide information as prompted by the Slack workflow.
- [ ] Link all artifacts **ONLY** in the Midpoint Review artifacts section below at least two days before the scheduled Midpoint Review.  **Do NOT add artifacts to Comments section** 

**Midpoint Review artifacts**

See Platform guidance on [Midpoint Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review#Midpointreview-Artifacts). Platform feedback is based solely on the artifacts provided, as reviewed during the two days before the Midpoint Review meeting. Any work not included in the artifacts below or any ongoing work taking place during the review period may not be reflected in that feedback.

Provide links or documents for the following:

- [ ] Finalized design prototype or mockup
- [ ] Specify which pages are included in the review 
- [ ] Research plan
- [ ] Conversation guide
 
**Content artifacts**
  
- [ ] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) feedback you received, including a content source of truth, any relevant static content page and entry point updates, and the intake form.

**IA artifacts**

- [ ] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-caia-intake-request) feedback you received, including an IA review document or intake form.

##### Platform actions
- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C06JV0KHZNF/p1713214486307419)
- [x] Meeting date/time: Tuesday, April 23, 2024 at 11:30 a.m. ET

##### Platform actions

* Accessibility
  * [ ] Feedback ticket attached
  * [ ] No feedback
* Content
  * [ ] Feedback ticket attached
  * [ ] No feedback
* Design
  * [ ] Feedback ticket attached
  * [ ] No feedback
* IA
  * [ ] Feedback ticket attached
  * [ ] No feedback
* [ ] Update this ticket with the Zoom recording
  - Recording link
  - Passcode: 
  
##### VFS actions
  
- [ ] Review feedback tickets and comment on the ticket if there are any questions or concerns
  
</details>

### Analytics Request
<details>
  <summary>Toggle Analytics Request</summary>

#### VFS actions
- [ ] Complete the [analytics request process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html)
- [ ] Link to the Analytics Implementation and QA Request ticket below

</details>

### Contact Center Review
<details>
  <summary>Toggle Contact Center Review</summary>

#### VFS actions
- [ ] Complete the [Contact Center review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html)
- [ ] Link to the Contact Center review request ticket below

</details>

### Staging Review
<details>
  <summary>Toggle Staging Review</summary>
  
##### VFS actions
- Navigate to reference link: [Staging Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-review)  
- [x] Schedule your Staging Review when ready:
   - Open the [Calendly staging review calendar](https://calendly.com/collaboration-cycle/staging-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [x] Notify the Collaboration Cycle team of this event by using the `/Collab Cycle Staging Review` "slash-command" in your team channel. Provide information as prompted by the Slack workflow.
- [ ] If this product contains any [experimental design](https://design.va.gov/experimental-design/), add the `experimental-design` label and schedule a meeting with DSC to present the research findings.
- [x] Link all artifacts **ONLY** in the Staging Review artifacts section below at least four days before the scheduled Staging Review.  **Do NOT add artifacts to Comments section**
- [ ] I confirm the environment is available and test users have been provided. <<<< **Will provide test users by EOD 4/16**
- [x] Please verify your product information in the [Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory).
 
**Staging Review artifacts**
  
See Platform guidance on [Staging Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html#Stagingreview-Artifacts). Platform findings are based solely on the staging URL and test users provided, as reviewed during the four days before the Staging Review meeting. Any test cases not covered or any ongoing work taking place during the review period may not be reflected in those findings.  

**Product artifacts** 
- [x] Staging URL: **staging.va.gov/profile/direct-deposit**
- [x] Specify which pages are included in the review  **(staging.va.gov/profile/direct-deposit < new unified form)** the toggle that must be enabled for this to work as expected is: **(profile_show_direct_deposit_single_form)**    
- [x] [Staging test user information    ](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/direct_deposit/ch33-bdn-corpdb-migration/qa/direct-deposit-ch33-test-accounts.md)     
  
**Note:** Please double-check that you've provided staging access information appropriate for testing the tool or feature. Don't put staging credentials in your va.gov-team ticket; store [test user information, passwords, and tasks](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-accessible-example.md) in a .md file in the va.gov-team-sensitive repository.

**Content artifacts**
- [x] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) feedback you received, including a content source of truth, any relevant static content page and entry point updates, and the intake form.
  
> [!NOTE]
> Content feedback was given directly on our Figma Files

- [current user flow](https://www.figma.com/file/CUR39JNnF2CS8SidGiWmYG/Profile---Direct-Deposit?type=design&node-id=0%3A1&mode=design&t=LLbs8brafQcokSvv-1)
- [vs proposed user flow after the updates/migration](https://www.figma.com/file/CUR39JNnF2CS8SidGiWmYG/Profile---Direct-Deposit?type=design&node-id=347-7871&mode=design&t=LLbs8brafQcokSvv-4)

**IA artifacts**
- [x] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-caia-intake-request) feedback you received, including an IA review document or intake form.

> [!NOTE]
> This isn't applicable since we're modifying a page that already exists to remove one form. Essentially switching from two forms to one. 

**QA artifacts**

An artifact that corresponds to each standard in the [QA Standards](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards).
- [x] [Regression test plan](https://docs.google.com/spreadsheets/d/1rOD8w0YNTlfbefVK7t2RsWoYK_ui1gTkVhCXlJE5VDg/edit#gid=409494960)
- [x] [Test plan](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/direct-deposit/ch33-bdn-corpdb-migration/qa)
- [ ] [Coverage for References](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards#QAstandards-traceability-reportsTraceabilityReports)
- [x] [Summary(Defects) reports](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/ch33-bdn-corpdb-migration/qa/README.md)
- [x] [E2E tests](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards#QAstandards-e2e-test-participationE2ETestParticipation) Main E2E tests for direct deposit work: https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/personalization/profile/tests/e2e/direct-deposit - All E2E tests not in the 'legacy' folder are new tests around this work and cover both happy paths and error cases
- [x] [Code coverage](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards#QAstandards-unit-test-coverageUnitTestCoverage) Component unit tests: https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/personalization/profile/tests/components/direct-deposit - Just like the E2E tests, all unit tests not within the legacy folder are new tests. There are also tests for the useDirectDeposit hook here: https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/personalization/profile/tests/hooks/useDirectDeposit

Current unit test coverage for the profile is +80%
![profile-coverage](https://github.com/department-of-veterans-affairs/va.gov-team/assets/8332986/9634aa14-82d4-49eb-a196-610f0a6eb7fb)

  
**Accessibility artifacts**
  
- [ ] [Completed accessibility testing artifact](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-testing.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5D). For details, see [Prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review).
  
##### Platform actions
- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C06JV0KHZNF/p1713216761824309?thread_ts=1713214486.307419&cid=C06JV0KHZNF)
- [x] Meeting date/time: Tuesday, April 23, 2024 at 11:30 a.m. ET.

##### Platform actions

- [x] Update this ticket with the Zoom recording
  - [Staging Review recording](https://oddball-io.zoom.us/rec/share/VUX1-fcPwmNKi2QV5xLUB0a3R5lQZ29Ve0FBjL_74siKDGYZIp2_jim2PMNnQAIu.IOeY7oenOOKuJvAK)
  - Password: `e=+9ijy@`
  
##### VFS actions
  
- [ ] Review the findings tickets and comment on the ticket if there are any questions or concerns
- [ ] Close individual findings tickets when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment on the issue ticket.
- [ ] After launch, [request an accessibility audit from the VA 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/request-support-from-the-va-508-office#RequestsupportfromtheVA508office-AuditRequest). This is required even if no accessibility issues were found during the Staging Review.
  - [ ] Share ServiceNow ticket number here: ______
- [ ] Close ticket once Privacy, Security, Infrastructure Readiness Review has been completed, VA 508 Office audit is requested, and all other post-Staging actions are complete
</details>

### Privacy, Security, Infrastructure Readiness Review
<details>
  <summary>Toggle Privacy, Security, Infrastructure Readiness Review</summary>
  
#### VFS actions
- [ ] Complete the [Privacy, security, infrastructure readiness review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Privacy-and-security-review.1782317101.html)
- [ ] Link to the Readiness Review ticket below

#### Platform actions
- [ ] Privacy, security, infrastructure readiness review is complete

</details>

**Comment by mtcA6** [comment]
@shiragoodman we've deployed everything and relaunched the CH33 DD page. I want to close this out even though there is one item tied to this ticket. 

That one item is open because we submitted for an experimental design. I'm tracking that ticket so I don't think we need this Collab Cycle ticket open as well. Is it okay to close this out? 
![image](https://github.com/user-attachments/assets/a9c6ba16-96c6-41bc-9d41-1e70a6f73668)
